packages/tooltip/test/lion-tooltip-arrow.test.js

```diff
@@ -1,87 +1,71 @@
 import { aTimeout, expect, fixture, html } from '@open-wc/testing';
+import '../../button/lion-button.js';
```

Comment:
Do we really need this dependency, or would a normal button be enough?

Reply:
this is because it's a div under the hood, because a native button has different dimensions on different browsers